frontend/app/templates/components/background-component/def-image-component.hbs

```diff
@@ -4,15 +4,33 @@
         <div class="default text">Select background from list</div>
         <i class="dropdown icon"></i>
         <div class="menu">
+            <br>
+            <div style="margin-left: 10px;" class="ui toggle checkbox listview">
+                    <input type="checkbox" name="public" onchange={{action 'change'}}>
+                <label>List view</label>
+            </div>
+            <br>
+            <br>
+            {{#if listview}}
             {{#each defImages as |image|}}
```

Comment:
Can you intend this so that it may not be of the same intend of that of the if clause.

Reply:
Done